24_find_lab/findlab.txt

```diff
@@ -1,24 +1,65 @@
-1. Write command lines to find using "find":
+1. Write command lines to find using "find":
+
 
   -. All files owned by yourself in /tmp folder
 
-  -. All files and directories not owned by you in /tmp folder
+  
+
+~/tmp find.
+
+-. All files and directories not owned by you in /tmp folder
+
+ 
+
+$ find . ! -user roi
+
+ -. All files, folders and links in your home folder whose name ends with a ".c"
 
-  -. All files, folders and links in your home folder whose name ends with a ".c"
+
+cd ~ find . -name '*.c'
+
 
   -. All files in your home folder whose name starts with an "a", "e", "i", "o" or "u"
 
-  -. All files in your home directory whose size is above 10MB
+  
+
+find . -name "*[aeiou]"
+
+-. All files in your home directory whose size is above 10MB
+
+  
+
+find . -size +10M
+
+-. All files in your home directory which were modified in the past 2 days
+
+
+
+find/home/roi -mtime -2
+
+
+
+  -. All files in your home directory which were not modified for over a month, and size is above 10MB.
+
+find . -mtime +30 -size +10M
+
+
 
-  -. All files in your home directory which were modified in the past 2 days
 
-  -. All files in your home directory which were not modified for over a month, and size is above 10MB.
+2. Create a hard link to a file of your choice and put it in a subdirectory under your home folder. How can you use "find" to locate the hard link given the original file?
+
 
-2. Create a hard link to a file of your choice and put it in a subdirectory under your home folder. How can you use "find" to locate the hard link given the original file?
+ln "music" "link-to-music"
+ls -l "music"
+mkdir secret
+mv music /home/secret ---> premission denied 

```

Comment:
`/home` is not your home directory. Your home directory is `/home/rpi`, and you can always get there by typing `cd`. Also the command `echo $HOME` will always print the full path to your home directory